data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelLogsDecoder.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  * Copyright OpenSearch Contributors
+ *   * SPDX-License-Identifier: Apache-2.0
+ *    */
+
+package org.opensearch.dataprepper.plugins.otel.codec;
+
+import io.opentelemetry.proto.collector.logs.v1.ExportLogsServiceRequest;
+import org.opensearch.dataprepper.model.codec.ByteDecoder;
+import org.opensearch.dataprepper.model.record.Record;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.log.OpenTelemetryLog;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.function.Consumer;
+
+
+public class OTelLogsDecoder implements ByteDecoder {
```

Comment:
Is this class for decoding data for unit test ?

Reply:
Not for unit test. It is for use by Kafka Buffer